.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Feature Request
+about: Suggest a new feature or enhancement for Chorus
+title: '[FEATURE] '
+labels: enhancement
+assignees: ''
+
+---
+
+## Feature Request
+
+### Description
+A clear and concise description of the proposed feature.
+
+### Backwards compatibility
```

Comment:
It is useful, however a generic user might not be aware of inner guts of chorus.
I'd move it lower.

Reply:
done